src/observers/languageServerManager.ts

```diff
@@ -0,0 +1,18 @@
+import { BaseEvent } from '../common/loggingEvents';
```

Comment:
consider moving this file to the `build` (which will be renamed to `validation` in the future as we talked)